File: [프로그래머스] 코딩테스트 입문_겹치는 선분의 길이.py
def solution(lines):
    answer = 0
    setArr = [0 for _ in range(200)]
    for i in lines:
        for j in range(i[0] ,i[1]):
            setArr[j + 100] += 1
    return setArr.count(2) + setArr.count(3)

File: [프로그래머스] 코딩테스트 입문_겹치는 선분의 길이.js
function solution(lines) {
    let answer = Array(200).fill(0);
    let cnt = 0;
    for(let i of lines){
        for(let j = i[0]; j < i[1]; j++){
            answer[j + 100]++;
            if(answer[j + 100] === 2){ cnt++; }
        }
    }
    return cnt;
}